rt/vm/longs/src/main/scala/org/apidesign/bck2brwsr/vm4brwsr/Module.scala

```diff
@@ -0,0 +1,46 @@
+/**
+ * Back 2 Browser Bytecode Translator
+ * Copyright (C) 2012-2017 Jaroslav Tulach <jaroslav.tulach@apidesign.org>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. Look for COPYING file in the top folder.
+ * If not, see http://opensource.org/licenses/GPL-2.0.
+ */
+package org.apidesign.bck2brwr.vm4brwsr
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+import scala.scalajs.runtime.RuntimeLong
+
+@JSExportAll
+@JSExportTopLevel( "Longs" )
+object Module {
```

Comment:
This file is certainly easier to manage than the original `java_lang_Number.js`.